DegreeProject/Assets/Scripts/Player/Player.cs

```diff
@@ -7,6 +7,7 @@
 public class Player : NetworkBehaviour
```

Comment:
För spelaren vill du nog spana in hur du gjorde med AIn och uppdatera din movement/animationskod, så allt syncas bra över nätverk.